TimetableApp/TimetableApp.Shared/ViewModels/HomePageViewModel.cs

```diff
@@ -122,10 +126,21 @@ private async Task Periodic()
 
         private void SetCurrentLesson(Lesson newLesson)
         {
-            if (currentLesson != newLesson)
+            
+            if (currentLesson != newLesson) 
             {
+                
                 currentLesson = newLesson;
                 if (DoAutoJoin) AutoJoin();
+                if (currentLesson != null)
+                {
+                    var notification = new ToastNotification();
+                    notification.Title = "New Lesson";
+                    notification.Message = $"Next Lesson: {currentLesson.Subject} start on {currentLesson.StartTime}";
```

Comment:
[Grammar Nazi]
I suggest changing the text to: `$"Lesson {currentLesson.Subject} has started on {currentLesson.StartTime}"`

Reply:
okay